content/en/docs/concepts/cluster-administration/cloud-providers.md

```diff
@@ -52,6 +52,12 @@ for the [kube-apiserver](/docs/admin/kube-apiserver/), [kube-controller-manager]
 For all external cloud providers, please follow the instructions on the individual repositories,
 which are listed under their headings below, or one may view [the list of all repositories](https://github.com/kubernetes?q=cloud-provider-&type=&language=)
 
+## Alibaba Cloud
+ If you wish to use the external cloud provider, its repository is [kubernetes/cloud-provider-alibaba-cloud](https://github.com/kubernetes/cloud-provider-alibaba-cloud)
```

Comment:
Do you have a guide or tutorial on how to set up the external provider? If so I would suggest linking it here as well.